vector-im/element-android
# Increase max length of voice messages to 15m (PSG-663)

Relates to: vector-im/element-ios#5415
Relates to: https://github.com/vector-im/element-android/issues/4160

Signed-off-by: Johannes Marbach <johannesm@element.io>


 
## Type of change


## Content

This raises the recording limit for voice messages from 2 to 15 minutes.

## Motivation and context

vector-im/element-ios#5415

## Tests

- Start recording a voice message
- Lock the recording or keep holding the button past 2 minutes
- See that the recording is still running

## Tested devices

- OS version(s): 12

## Checklist
